cucumber/authenticators_oidc/features/authn_oidc.feature

```diff
@@ -35,8 +44,10 @@ Feature: OIDC Authenticator - Hosts can authenticate with OIDC authenticator
       role: !group conjur/authn-oidc/keycloak/users
       member: !user alice
     """
-    And I am the super-user
-    And I successfully set OIDC variables
+    And I set the following conjur variables:
```

Comment:
@jvanderhoof - Do we need to add variable or note to feature description that this is run as admin user?

Reply:
@adamouamani, ironically, the line `I am the super-user` didn't actually do anything. Policy and variables are loaded using the `admin` user.  There are a couple of cases (the scenarios which continue to use that line), where we need the line (typically when accessing something related to the file system).

Reply:
@jvanderhoof as we were discussing, these steps should be more declarative and reusable.  The line about `I am the super-user` seems like something for background describing persona.  I'm also wondering how we can use your pattern of data tables and scenario context to make steps definitions more reusable and flexible